core/trino-main/src/test/java/io/trino/execution/TestSqlTaskManager.java

```diff
@@ -264,6 +265,10 @@ public void testFailStuckSplitTasks()
         // Here we explicitly enqueue an indefinite running split runner
         taskExecutor.enqueueSplits(taskHandle, false, ImmutableList.of(mockSplitRunner));
         taskExecutor.start();
+        // wait for the task executor to start processing the split
+        while (!mockSplitRunner.isStarted()) {
```

Comment:
You can call `get` on the `started` future (ideally with some reasonable timeout, e.g.: 10 seconds) instead of looping

Reply:
thanks!